dojo/finding/views.py

```diff
@@ -1936,6 +1968,31 @@ def finding_bulk_update_all(request, pid=None):
                         # it will trigger the pre_save signal
                         find.save_no_options()
 
+                        if system_settings.false_positive_history:
+                            # If finding is being marked as false positive
+                            if find.false_p:
+                                do_false_positive_history(find)
+
+                            # If finding was a false positive and is being reactivated: retroactively reactivates all equal findings
+                            elif old_find.false_p and not find.false_p:
+                                if system_settings.retroactive_false_positive_history:
+                                    logger.debug('FALSE_POSITIVE_HISTORY: Reactivating existing findings based on: %s', find)
+
+                                    existing_fp_findings = match_finding_to_existing_findings(
+                                        find, product=find.test.engagement.product
+                                    ).filter(false_p=True)
+
+                                    for fp in existing_fp_findings:
+                                        logger.debug('FALSE_POSITIVE_HISTORY: Reactivating false positive %i: %s', fp.id, fp)
+                                        fp.active = find.active
+                                        fp.verified = find.verified
+                                        fp.false_p = False
+                                        fp.out_of_scope = find.out_of_scope
+                                        fp.is_mitigated = find.is_mitigated
+                                        fp.last_reviewed = timezone.now()
+                                        fp.last_reviewed_by = request.user
```

Comment:
Same comment as ^

> This might get a little confusing as it would create instances of findings being "reviewed" by someone who may not have any knowledge of the findings existence